CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.49.1
+==========
+### Bug Fixes
```

Comment:
```suggestion
### New Features

Extended Managed Identity experimental functionality with support for Azure Arc.

### Bug Fixes
```

Reply:
We did not include that merge commit in the release build we did

Reply:
Ignore the comment as now we are taking the main branch for the release 

@pmaytak FYI